lib/handler.js

```diff
@@ -32,9 +32,8 @@ export const createHandler = ({
  * @param {import('pg').Pool} pgPool
  */
 const handler = async (req, res, pgPool) => {
-  // Fix the edge case: new URL('//foo', 'http://127.0.0.1') produces href "http://foo/"
-  const reqUrl = req.url.replace(/^\/+/, '/')
-  const { pathname, searchParams } = new URL(reqUrl, 'http://127.0.0.1')
+  // Caveat! `new URL('//foo', 'http://127.0.0.1')` would produce "http://foo/" - not what we want!
```

Comment:
What do you think about removing the comment and adding a test for this instead? To me that's the programming native version of this solution

Reply:
We already have a test in place, I added it in #6.

https://github.com/filecoin-station/spark-stats/blob/11fc6b1d8f96a9ff855258131620409b3e26ff56/test/handler.test.js#L54-L57

I suppose I can add another test to verify now we handle an existing route with double-slash.

I prefer to keep the comment to remind readers why we are using string concatenation instead of `new URL(path, base)`. In my experience, it's better to be explicit.